irohad/main/server_runner.hpp

```diff
@@ -14,14 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include <grpc++/grpc++.h>
 #include <grpc++/server_builder.h>
 
-namespace connection {
+#ifndef CONNECTION_SERVER_RUNNER_HPP
+#define CONNECTION_SERVER_RUNNER_HPP
 
-    class ServerRunner {
-    public:
-        ServerRunner(const std::string& ip,
-                     const std::vector<grpc::Service*>& services);
-    };
+namespace server_runner {
+  void initialize(const std::string &ip, int port,
+                  const std::vector<grpc::Service *> &services);
+  void run();
```

Comment:
For clarify i write remark. No so good make classes as "hard" implementations. We should care about testability of classes. One of main approaches to do it is separate interface and implementation. In terms of c++ it is make abstract class with all abstract method (interface), make class with implementation that extends interface. And all usages of class should be provided by interface.